db/schema.rb

```diff
@@ -60,16 +60,20 @@
 
   create_table "items", force: :cascade do |t|
     t.integer  "unit_price"
-    t.boolean  "pending",         default: false
-    t.datetime "created_at",                      null: false
-    t.datetime "updated_at",                      null: false
-    t.boolean  "sold",            default: false
+    t.boolean  "pending",             default: false
+    t.datetime "created_at",                          null: false
+    t.datetime "updated_at",                          null: false
+    t.boolean  "sold",                default: false
     t.string   "section"
     t.string   "row"
     t.string   "seat"
     t.string   "delivery_method"
     t.integer  "user_id"
     t.integer  "event_id"
+    t.string   "ticket_file_name"
+    t.string   "ticket_content_type"
+    t.integer  "ticket_file_size"
+    t.datetime "ticket_updated_at"
   end
```

Comment:
Is rails smart enough that if you have an attachment type to make all of these columns? pretty cool!